journal-of-microbiology-and-biotechnology.csl

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-GB">
+  <info>
+    <title>Journal of Microbiology and Biotechnology</title>
+    <title-short>JMB</title-short>
+    <id>http://www.zotero.org/styles/journal-of-microbiology-and-biotechnology</id>
+    <link href="http://www.zotero.org/styles/journal-of-microbiology-and-biotechnology" rel="self"/>
+    <link href="http://www.zotero.org/styles/embo-reports" rel="template"/>
```

Comment:
Was this derived from embo-press.csl? https://github.com/citation-style-language/styles/blob/master/dependent/embo-reports.csl is a dependent style.

Reply:
At least, it has been since last September: https://github.com/citation-style-language/styles/commit/ef4d15e1da0f5823bf805dc2a0938c882c7ef650